### PR TITLE
List (mobile): Fix to apply the GUI feedback

### PR DESCRIPTION
**Pull request by HunseopJeong**
[Issue] https://github.com/Samsung/TAU/issues/980
[Problem] Need to change some styles according to GUI feedback
[Solution]
- Expandable List: Adjust the position of the toggle icon
- Multiple Selection List: Change the margin of the ui-btn-icon when the another button is placed before the icon button

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

File: src/css/profile/mobile/common/listview.less [hunk 1]
```diff
@@ -173,6 +173,13 @@ tau-expandable {
 					width: 15 * @px_base;
 					height: 15 * @px_base;
 				}
+
+				// line height between title and sub
```
Comment (singa2000) [inline src/css/profile/mobile/common/listview.less]: I have a question.
It was already .ui-li-text-sub set the height=4*px.
Isn't it duplicated?

				// line height between title and sub
				&+.ui-li-text-sub::before {
					content: "";
					display: block;
					height: 4 * @px_base;
				}

Comment (HunseopJeong) [reply src/css/profile/mobile/common/listview.less]: @singa2000,
You are right. I uploaded wrong commit.
Thank you.

**Review by singa2000 — APPROVED** [review]
Looks good to me.

File: src/css/profile/mobile/common/appbar.less [hunk 2]
```diff
@@ -222,6 +226,7 @@
 	}
 
 	.ui-label-select-all {
+		.font(regular);
```
Comment (singa2000) [inline src/css/profile/mobile/common/appbar.less:229]: Would you check the  3px padding between checkbox and "All" text,, please?

Comment (HunseopJeong) [reply src/css/profile/mobile/common/appbar.less:229]: Thank you.
It is the part I didn't check.
Done.

**Review by singa2000 — APPROVED** [review]
Good

**Pull request merged**